### PR TITLE
add unfinished draft announcement

### PR DESCRIPTION
**Pull request by bbrcknl**

### CODE REVIEW

**Comment by bbrcknl** [comment]
Re link errors:
https://enrollment.lfx.linuxfoundation.org/?project=seL4 is down for scheduled maintenance

I can get to both https://www.autoware.org/ and https://defcon.org/html/defcon-29/dc-29-index.html. I've had these issues before, where the link checker fails but I can click on the links.

File: content/Foundation/Summit/2023/home.pml [hunk 1]
```diff
@@ -15,11 +15,12 @@ Minneapolis, USA
 
 <div class="summit_div">
   <ul>
-    <li><a href="#summit-news">Latest News: Register for the summit</a></li>
+    <li><a href="#summit-news">Latest News: Keynotes announcedRegister for the summit</a></li>
```
Comment (june-andronick) [inline content/Foundation/Summit/2023/home.pml:18]: Typo: remove "Register for the summit"

File: content/Foundation/Summit/2023/home.pml [hunk 2]
```diff
@@ -34,34 +35,40 @@ Minneapolis, USA
     </h3>
   </div>
   <h3 class="highlight">
-    Register for the summit
+    Keynotes announced
   </h3>
   <p>
-    The seL4 summit will cover the complete seL4 ecosystem, consisting of the verified microkernel, as well as all seL4-related technology, tools, infrastructure, products, projects, and people.
+    We are pleased to announce two keynotes for the seL4 summit 2023.
```
Comment (june-andronick) [inline content/Foundation/Summit/2023/home.pml:41]: I suggest:

We are pleased to announce that the two keynotes for the seL4 summit 2023 will be Gage from NCSC and Sam Leffler from Google!

~

Then add Gage's bio, followed by:

Gage will talk about "Scoping assurance activities with seL4". 

~
Then add Gage's abstract.

Then same for Sam: bio, talk title, abstract.

File: content/news/2023.pml [hunk 3]
```diff
@@ -55,6 +55,34 @@ Older News: <%=reflist%>
 -->
 
 
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="summit-2023-keynotes">
+  <a href="#summit-2023-keynotes"><span class="tag-span"># </span></a> 19 June 2023: Keynotes for seL4 summit 2023 announced
+</div>
+<div class="row" style="margin-left:5px; margin-right:5px">
+  <p>
+    We are pleased to announce two keynotes for the seL4 summit 2023.
```
Comment (june-andronick) [inline content/news/2023.pml:64]: Change as above, but shorter (no abstracts) and add links to summit page.

We are pleased to announce that the two keynotes for the [seL4 summit 2023](link to summit page) will be Gage from NCSC and Sam Leffler from Google! Gage will talk about ["Scoping assurance activities with seL4"](link to abstract in summit page) and Sam about ["CantripOS: An OS for Ambient ML Applications"](link to abstract in summit page). 

~
Then bios.

Also: in this news item, I'd change the pictures to be 2 logos: NCSC and Google. 

And I'd add links to the logos (going to NCSC and google)

File: content/Foundation/Summit/2023/home.pml [hunk 4]
```diff
@@ -34,34 +35,40 @@ Minneapolis, USA
...
-    <li>Networking with other seL4 experts and enthusiasts</li>
-    <li>Reception and Dinner</li>
-  </ul>
+  </div>
+  <div class="row" style="margin-left:5px; margin-right:5px">
   <p>
-    <strong><a href="https://events.linuxfoundation.org/sel4-summit/">Register here</a></strong>
+    <img src="../../../images/summit/sam-security-photo.jpg"
+    style="width: 15%; float:right; margin-left:100px"
+    alt="Sam Leffler" />
```
Comment (june-andronick) [inline content/Foundation/Summit/2023/home.pml:55]: For this summit page, I'd but both pictures (NCSC logo and Sam's picture) side by side centered (as we had before for Gernot and Boyd, or the panelists). It'll also avoid the impression Sam is from NCSC..

**Review by june-andronick — COMMENTED** [review]
Thanks @bbrcknl . I've added a number of comments/suggestions.

**Review by june-andronick — APPROVED** [review]

**Pull request merged**